types/internal-slot/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for internal-slot 1.0
+// Project: https://github.com/ljharb/internal-slot#readme
+// Definitions by: ExE Boss <https://github.com/ExE-Boss>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/**
+ * Asserts that `O` is an `object`, `slot` is a `string`
+ * and `O` has the internal slot `slot`.
+ */
+export function assert(O: object, slot: string): void;
```

Comment:
```suggestion
export function assert(O: object, slot: string): void | never;
```
perhaps? also this should use that new syntax that indicates that it can throw, if possible

Reply:
That just collapses into `void`.